CustomBatchWatchdog.cs

```diff
@@ -21,8 +21,8 @@ public partial class CustomBatchWatchdog : ServiceBase
         List<RecoveryItem> recoveryItems = new List<RecoveryItem>();
 
         // local constants (not configurable)
```

Comment:
remove this comment, they are now configurable

Reply:
Done